src/pymor/reductors/basic.py

```diff
@@ -295,6 +323,101 @@ def build_rom(self, projected_operators, error_estimator):
                                  error_estimator=error_estimator, **projected_operators)
 
 
+class StationaryLSRBReductor(ProjectionBasedReductor):
+    """Least-squares projection based reductor for stationary problems.
+
+    This reductor uses least-squares projection instead of Galerkin projection.
+
+    Parameters
+    ----------
+    fom
+        The full order |Model| to reduce.
+    RB
+        The basis of the reduced space onto which to project. If `None` an empty basis is used.
+    product
+        Inner product |Operator| w.r.t. which `RB` is orthonormalized. If `None`, the Euclidean
+        inner product is used.
+    use_normal_equations
+        If `True`, solve the normal equations (A^* W A x = A^* b) instead of using a least-squares
+        solver. If `False`, equip the operator with a least-squares solver.
+    check_orthonormality
+        See :class:`ProjectionBasedReductor`.
+    check_tol
+        See :class:`ProjectionBasedReductor`.
+    """
+
+    def __init__(self, fom, RB=None, product=None, use_normal_equations=False,
+                 check_orthonormality=None, check_tol=None):
+        assert isinstance(fom, StationaryModel)
+        RB = fom.solution_space.empty() if RB is None else RB
+        assert RB in fom.solution_space
+        super().__init__(fom, {'RB': RB}, {'RB': product},
+                         check_orthonormality=check_orthonormality, check_tol=check_tol)
+        self.use_normal_equations = use_normal_equations
+        self.product = product
+
+    def project_operators(self):
+        fom = self.fom
+        RB = self.bases['RB']
+
+        if self.use_normal_equations:
+            X_h_inv = None
+            if self.product:
+                X_h_inv = InverseOperator(self.product)
+
+            # Solve normal equations: (A^* W A) x = A^* W b
```

Comment:
Harmonize notation (W = X_h_inv). Also, I got totally confused where the inverse is coming from. It would be good to leave a comment that we are assuming that A maps to the dual space.

Reply:
i added a short explanation.